packages/go_router/lib/src/go_router_state.dart

```diff
@@ -56,6 +56,59 @@ class GoRouterState {
   /// An extra object to pass along with the navigation.
   final Object? extra;
 
+  /// Handle extra object passed along with the navigation.
```

Comment:
```suggestion
  /// Handle extra object passed along with the navigation.
  ///
```

https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph